### PR TITLE
refactor: use next with /deployments pagination

### PR DESCRIPTION
**Pull request by nchamo**
Fixes https://github.com/decentraland/catalyst-client/issues/84

### CODE REVIEW

File: test/ContentClient.spec.ts [hunk 1]
```diff
@@ -308,40 +308,12 @@ describe('ContentClient', () => {
     expect(result).toEqual([deployment])
   })
 
-  it('When fetching all deployments with offset pagination, then the result is as expected', async () => {
+  it('When fetching all deployments with pagination, then subsequent calls are made correctly', async () => {
```
Comment (agusaldasoro) [inline test/ContentClient.spec.ts:311]: how are you testing that subsequent calls are made correctly?

File: src/ContentClient.ts [hunk 2]
```diff
@@ -260,52 +256,15 @@ export class ContentClient implements ContentAPI {
       queryParams.set('fields', [fieldsValue])
     }
 
-    // Set legacy filters for get all deployments
```
Comment (agusaldasoro) [inline src/ContentClient.ts:263]: Is it safe to delete this now? Or should we wait?

**Review by agusaldasoro — APPROVED** [review]
Just two comments

File: src/ContentClient.ts (hunk 2 above)
Comment (nchamo) [reply src/ContentClient.ts:263]: It's ok. All catalysts support `from` already, right?

File: test/ContentClient.spec.ts (hunk 1 above)
Comment (nchamo) [reply test/ContentClient.spec.ts:311]: Lol, good point. Just changed it

**Pull request merged**